### ISSUE TITLE
probably, useless or buggy removal of <UNK> in librispeech tdnnf training script

### ISSUE THREAD

**Issue by al-zatv** [kaldi-asr/kaldi#3476]
Hello! In script 
https://github.com/kaldi-asr/kaldi/blob/master/egs/librispeech/s5/local/chain/tuning/run_tdnn_1d.sh
where are lines:

```
graph_dir=$dir/graph_tgsmall
if [ $stage -le 16 ]; then
  # Note: it might appear that this $lang directory is mismatched, and it is as
  # far as the 'topo' is concerned, but this script doesn't read the 'topo' from
  # the lang directory.
  utils/mkgraph.sh --self-loop-scale 1.0 --remove-oov data/lang_test_tgsmall $dir $graph_dir
  # remove <UNK> from the graph, and convert back to const-FST.
  fstrmsymbols --apply-to-output=true --remove-arcs=true "echo 3|" $graph_dir/HCLG.fst - | \
    fstconvert --fst_type=const > $graph_dir/temp.fst
  mv $graph_dir/temp.fst $graph_dir/HCLG.fst
fi
```

As far as I understand,  "utils/mkgraph.sh ... --remove-oov" do the same as  "fstrmsymbols.... "echo 3|"" , so it is just waste of time. 
Also, in my modifications of librispeech code, <UNK> is at other index, not 3, so in my case it is also a bug.

If I understand this code right, it may have sence to remove "fstrmsymbols ..." part.

**Comment by danpovey** [comment]
yes, you seem to be right.  Let me bear it in mind for next time we update the Librispeech recipe, we can fix that the next time we add a new tuning script.  @ChunChiehChang remember this in case you are the next one to do this.

**Comment by danpovey** [comment]
Resolved in #3621
